src/core/model_repository_manager.cc

```diff
@@ -296,10 +296,10 @@ class ModelRepositoryManager::BackendLifeCycle {
   // Get the VersionStateMap representation of the specified model.
   const VersionStateMap GetVersionStates(const std::string& model_name);
 
-  // The the state of a specific model version.
+  // Get the states of all recorded model version.
```

Comment:
Does't this make it the same as "GetVersionStates", which is not what is needed for where it is used.